mockers/src/lib.rs

```diff
@@ -8,6 +8,8 @@ use std::rc::{Rc, Weak};
 use std::fmt::Write;
 use std::ops::DerefMut;
 
+use itertools::Itertools;
```

Comment:
What do you use from itertools?

Reply:
Describe was implemented as:

```rust
fn describe(&self) -> String {
    format!(
        "{}({}, {})",
        self.get_method_name(),
        self.arg0.describe(),
        self.arg1.describe()
    )
```
(with different number of `argN`s for each type)

Now it is

```rust
fn describe(&self) -> String {
    let args: &[&std::fmt::Display] = &[$(&self.$arg.describe()),*];
    format!("{}({})", self.get_method_name(), args.iter().format(", ")) 
}
```

Where `format` comes from itertool. It could be avoided by making the macro a bit more complex, but given that `mockers_derive` already depended on `itertools`, I didn't see that new dependency as a big deal. (Although right now `mockers` and `mockers_derive` don't use the same version of `itertools`, but I wanted feedback on this PR before suggesting any more change).